api/analysis/retirejs.go

```diff
@@ -61,18 +63,37 @@ func RetirejsStartAnalysis(CID string, cOutput string) {
 		return
 	}
 
-	if strings.Contains(cOutput, "ERROR_RUNNING_RETIREJS") {
-		errorOutput := fmt.Sprintf("Container error: %s", cOutput)
+	failedRunning := strings.Contains(cOutput, "ERROR_RUNNING_RETIREJS")
+	if failedRunning {
 		updateContainerAnalysisQuery := bson.M{
 			"$set": bson.M{
 				"containers.$.cResult": "error",
-				"containers.$.cInfo":   errorOutput,
+				"containers.$.cInfo":   "Internal error running NPM Audit.",
```

Comment:
`containers.$.cInfo` can be `errorOutput`, this function is related to RetireJS.

Reply:
Thanks for this comment, @mdjunior! As we have just spoken, `cOutput` is already in MongoDB and we do not need to update it once again. For `cInfo`, I will check if other functions still use `cOutput` as `errMsg` and refactor them! 🙃